src/browsers/index.ts

```diff
@@ -82,6 +84,36 @@ export class BrowserManager {
     return dataDirPath;
   };
 
+  public getVersionJSON = async () => {
+    const port = await getPort();
+    const config = new Config();
+    config.setPort(port);
```

Comment:
I'd like to avoid this. Calling `confi.setPort` will change the active port _everywhere_ that browserless listens on. Instead, we should:

- Create the new browser object like you do below.
- Just call `launch` with no options or an empty object.
- Get the browser's debugging port from `browser.wsEndpoint` since it's exposed there.

I'd also rename this method since it's only CDP that supports this and not Playwright, so `getCDPVersionJSON`